src/new-score.js

```diff
@@ -0,0 +1,18 @@
+async function newScore(name, scr) {
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0CasFsHb7LOJqeFKBD5E/scores/';
+
+  const method = {
+    method: 'POST',
+    body: JSON.stringify({
+      user: name,
+      score: scr,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  };
+  const request = new Request(requestURL);
+  await fetch(request, method);
+}
```

Comment:
- [ ] In this project, and going forward we are required to use ES6. You must have come across this in milestone one in the learning objectives. Please let's refactor all functions to be of ES6 syntax.